velox/common/file/File.cpp

```diff
@@ -129,7 +129,11 @@ uint64_t InMemoryWriteFile::size() const {
   return file_->size();
 }
 
-LocalReadFile::LocalReadFile(std::string_view path) : path_(path) {
+LocalReadFile::LocalReadFile(std::string_view path)
+    : executor_(std::make_unique<folly::IOThreadPoolExecutor>(
```

Comment:
there is no need to run this in the context of an event base thread.

Use folly::CPUThreadPoolExecutor instead?